src/model.rs

```diff
@@ -5,18 +5,35 @@ pub struct Meigen {
     pub id: u32,
     pub author: String,
     pub content: String,
+    pub loved_user_id: Vec<u64>
+}
+impl Meigen {
+    pub fn loves(&self) -> usize {
+        self.loved_user_id.len()
+    }
+
+    pub fn is_loving(&self, user_id: u64) -> bool {
+        self.loved_user_id.iter().any(|&id| id == user_id)
```

Comment:
I know you're already poisoned by iterator
```suggestion
        self.loved_user_id.contains(&user_id)
```

Reply:
yes i really am